core/src/kvs/lq_v2_fut.rs

```diff
@@ -157,9 +157,9 @@ async fn process_change_set_for_notifications(
 					"There are {} table mutations being prepared for notifications",
 					table_mutations.1.len()
 				);
-				for (i, mutation) in table_mutations.1.iter().enumerate() {
+				for (_i, mutation) in table_mutations.1.iter().enumerate() {
```

Comment:
Fixes this issue:
```
 building '/nix/store/9zx5xa8a0rjj9c7b9xj25hfpaj5fliiy-surreal-1.5.0+20240423.8b1ea0f.drv'...
error: builder for '/nix/store/9zx5xa8a0rjj9c7b9xj25hfpaj5fliiy-surreal-1.5.0+20240423.8b1ea0f.drv' failed with exit code 101;
       last 25 log lines:
       >    --> core/src/kvs/lq_v2_fut.rs:160:10
       >     |
       > 160 |                 for (i, mutation) in table_mutations.1.iter().enumerate() {
       >     |                      ^ help: if this is intentional, prefix it with an underscore: `_i`
       >     |
       >     = note: `#[warn(unused_variables)]` on by default
       >
```
https://github.com/surrealdb/surrealdb/actions/runs/8804728549/job/24165890620